examples/zxi.rs

```diff
@@ -1,8 +1,12 @@
+#[cfg(all(feature = "smt", feature = "zok_front"))]
 use circ::front::zsharp::{Inputs, ZSharpFE};
+
+#[allow(unused_imports)]
 use circ::front::Mode;
 use std::path::PathBuf;
 use structopt::StructOpt;
 
+#[allow(dead_code)]
```

Comment:
Perhaps, leave a comment with why this is allowed?

Reply:
This is mainly to help with the getting rid of the compiler warnings from `zsharp_path: PathBuf` and other fields. Should I just feature gate it instead?